### PR TITLE
fix: tem add rc-util to top level in codesandbox

### PR DESCRIPTION
**Pull request by BoyYangzai**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link
https://github.com/ant-design/ant-design/issues/42671

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    fix tem modify rc-util@latest in codesandbox       |
| 🇨🇳 Chinese |      临时修复官网 codesandbox  中 rc-util 为 latest   |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at d43496a</samp>

Fix import issues and add dependency for `CodePreviewer` component. This component allows users to see and interact with ant-design components in the documentation.

### <samp>🤖 Generated by Copilot at d43496a</samp>

* Import the type `AntdPreviewerProps` for the `CodePreviewer` component ([link](https://github.com/ant-design/ant-design/pull/42672/files?diff=unified&w=0#diff-54c90701ed595205ea2cd4286b1cd0ee8b2f01f77a23088e0108feda7af2d65fR18))
* Remove the redundant import of the same type ([link](https://github.com/ant-design/ant-design/pull/42672/files?diff=unified&w=0#diff-54c90701ed595205ea2cd4286b1cd0ee8b2f01f77a23088e0108feda7af2d65fL30))
* Add the dependency `rc-util` for the live preview of ant-design components ([link](https://github.com/ant-design/ant-design/pull/42672/files?diff=unified&w=0#diff-54c90701ed595205ea2cd4286b1cd0ee8b2f01f77a23088e0108feda7af2d65fR334))

### CODE REVIEW

**Comment by BoyYangzai** [comment]
可以临时加在官网上 保证官网上codesandbox 正常使用，完整修复待 codesandbox 官方

File: .dumi/theme/builtins/Previewer/CodePreviewer.tsx [hunk 1]
```diff
@@ -331,6 +331,7 @@ createRoot(document.getElementById('container')).render(<Demo />);
     main: 'index.js',
     dependencies: {
       ...dependencies,
+      'rc-util': 'latest',
```
Comment (afc163) [inline .dumi/theme/builtins/Previewer/CodePreviewer.tsx:334]: 别用 latest，用 package.json 里的 rc-util 版本。

Comment (BoyYangzai) [reply .dumi/theme/builtins/Previewer/CodePreviewer.tsx]: 🫡

File: .dumi/theme/builtins/Previewer/CodePreviewer.tsx [hunk 2]
```diff
@@ -331,7 +331,7 @@ createRoot(document.getElementById('container')).render(<Demo />);
     main: 'index.js',
     dependencies: {
       ...dependencies,
-      'rc-util': 'latest',
+      'rc-util': '^5.32.0',
```
Comment (afc163) [inline .dumi/theme/builtins/Previewer/CodePreviewer.tsx:334]: 可以读一下 package.json

Comment (BoyYangzai) [reply .dumi/theme/builtins/Previewer/CodePreviewer.tsx]: done

File: .dumi/theme/plugin.ts [hunk 3]
```diff
@@ -3,6 +3,7 @@ import type { IApi, IRoute } from 'dumi';
 import ReactTechStack from 'dumi/dist/techStacks/react';
 import fs from 'fs';
 import sylvanas from 'sylvanas';
+import pkg from '../../package.json';
```
Comment (li-jia-nan) [inline .dumi/theme/plugin.ts:6]: 这个命名可以优化一下，但是好像不能直接叫 package，`package` 在严格模式中是个保留字，可以参考一下其它文件夹中的命名，叫 `localPackage`

Comment (BoyYangzai) [reply .dumi/theme/plugin.ts:6]: done

**Review by afc163 — APPROVED** [review]

**Pull request merged**